CompositeUi/src/views/page/Dashboard.js

```diff
@@ -14,18 +14,38 @@ import {connect} from 'react-redux';
 
 import Button from './../component/Button';
 import ContentMiddleLayout from './../layout/ContentMiddleLayout';
+import DashboardActions from './../../actions/Dashboard';
 import DashboardWidget from './../component/DashboardWidget';
 import FormInput from './../component/FormInput';
+import LoadingSpinner from './../component/LoadingSpinner';
 import LogoHeader from './../component/LogoHeader';
-import ProjectPreview from './../component/ProjectPreview';
+import ResourcePreview from './../component/ResourcePreview';
 import {Row, RowColumn} from './../component/Grid';
 
-@connect(state => ({projects: state.projects.projects}))
-class Index extends React.Component {
+@connect(state => ({
+  waiting: state.dashboard.fetching,
+  organizations: state.dashboard.organizations,
+  projects: state.dashboard.projects
+}))
```

Comment:
I believe is better to do it `state => ({dashboard: state.dashboard})`.

And use object destructing below instead of using the full `this.props.organization` for example